zipkin-storage/cassandra/src/test/java/zipkin/storage/cassandra/CassandraSpanConsumerTest.java

```diff
@@ -96,8 +96,8 @@ public void skipsRedundantIndexingInATrace() {
     });
 
     accept(trace);
-    assertThat(rowCount(Tables.SERVICE_SPAN_NAME_INDEX)).isEqualTo(4L);
-    assertThat(rowCount(Tables.SERVICE_NAME_INDEX)).isEqualTo(4L);
+    assertThat(rowCount(Tables.SPAN_DURATION_INDEX)).isGreaterThanOrEqualTo(4L);
```

Comment:
if possible, I'd tighten this up, as 20 (in the later assertion) is also greaterThanOrEqualTo(4)

or put a TODO and I can look into it post merge